### ISSUE TITLE
Different install results in different locations

### ISSUE THREAD

**Issue by JoshSchreuder** [bower/bower#2335]
**Output of `bower -v && npm -v && node -v`:**

```
λ bower -v && npm -v && node -v
1.7.9
3.7.1
v5.5.0
```

**Additional environment details (proxy, private registry, etc.):**
N/A

**Steps to reproduce the issue:**
1. Run `bower install ng-grid#2.0.14`
2. In different environments / location we get different results

Australia:
![](https://i.imgur.com/UduyOyw.png)

Pune:
![](http://i.imgur.com/HTduU2r.png)
1. Results of `bower info ng-grid#2.0.14` is different depending on environment / location

Australia:
![](https://i.imgur.com/OxodY6G.png)

Pune:
![](http://i.imgur.com/c2D4OrW.png)

**Describe the results you received:**

Oddly enough, files on disk seem to be the latest version in Pune (ie. master), but with a 2.0.14 bower.json somehow. 

In Australia, the correct tag is fetched and files on disk / directory structrue are all correct.

**Describe the results you expected:**

Results should be the same for the same command being run.

**Additional information:**

Any ideas on what potential differences may be? We're running the same codebase, same version of Bower etc.

The only difference I can see is location, but maybe I'm missing something.

**Comment by BenMann** [comment]
You seem to be on Windows and Unix respectively, which maaay cause differences. The logs also suggest that on both machines the package was cached by Bower, so clearing that is a good first step, if you haven't. Is there an actual difference in files after the installs? Also having the same npm/node versions could eliminate some shenanigans.

**Comment by maingi4** [comment]
Hey, I was the windows guy in the example... yes, the final files were different, I was getting the latest version while @JoshSchreuder was getting the correct version which was intended.

Josh had tried clearing the cache to no avail, I didn't try yesterday as yesterday was the first time I had installed bower on that machine and didn't imagine caching issues.

It stands solved as of now, I saw the same behaviour in another colleague's machine (also windows)... would love to understand what is going on.
